cmd/aro/portal.go

```diff
@@ -142,7 +142,7 @@ func portal(ctx context.Context, log *logrus.Entry) error {
 	}
 
 	clientID := os.Getenv("AZURE_PORTAL_CLIENT_ID")
-	verifier, err := middleware.NewVerifier(ctx, _env.TenantID(), clientID)
+	verifier, err := middleware.NewVerifier(ctx, _env, _env.TenantID(), clientID)
```

Comment:
nit: no longer need to pass _env.TenantID() if you're passing _env

Reply:
changed